Tasks/PublishBuildArtifacts/task.json

```diff
@@ -12,27 +12,19 @@
   "version": {
     "Major": 1,
     "Minor": 0,
-    "Patch": 9
+    "Patch": 0
   },
   "demands": [
   ],
-    "minimumAgentVersion": "1.83.0",
+    "minimumAgentVersion": "1.91.0",
   "inputs": [
     {
-      "name": "CopyRoot",
+      "name": "Content",
```

Comment:
Call it ArtifactContents and make the description Folder containing files that you want to include in the artifact